test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua

```diff
@@ -0,0 +1,68 @@
+---------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1912
+-- Description
+-- SDL must respond NACK in case navigation app connected over protocol v2 sends StartService for audio service
+-- Preconditions
+-- SDL and HMI are started.
+-- Steps to reproduce
+-- navigation app connects over protocol v2 and this app sends StartService for audio service
+-- Actual result
+-- N/A
+-- Expected result
+-- SDL must respond StartService_NACK to this app
+---------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local constants = require('protocol_handler/ford_protocol_constants')
+local commonDefects = require('test_scripts/Defects/4_5/commonDefects')
+local events = require('events')
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
+
+--[[ Local Functions ]]
+local function StartNaviServiceVia2Protocol(self)
+  self.mobileSession1:Send({
```

Comment:
@GetmanetsIrina Taking to account asynchronous event/expectation system of ATF I would propose to send start session request after creation and initialization of all events for response expectation but before mentioned expectation registering.